src/impl/util.js

```diff
@@ -208,18 +216,20 @@ export function signedOffset(offHourStr, offMinuteStr) {
 
 // COERCION
 
-export function normalizeObject(obj, normalizer, ignoreUnknown = false) {
+function asNumber(value) {
+  const numericValue = Number(value);
+  if (typeof value === "boolean" || value === "" || Number.isNaN(numericValue))
+    throw new Error(`Invalid unit value ${value}`);
+  return numericValue;
+}
+
+export function normalizeObject(obj, normalizer, nonUnitKeys) {
   const normalized = {};
   for (const u in obj) {
     if (obj.hasOwnProperty(u)) {
       const v = obj[u];
-      const numericValue = Number(v);
-      if (v !== null && !Number.isNaN(numericValue)) {
-        const mapped = normalizer(u, ignoreUnknown);
-        if (mapped) {
-          normalized[mapped] = numericValue;
-        }
-      }
+      if (v === undefined || v === null || nonUnitKeys.includes(u)) continue;
```

Comment:
I think you meant to check on `u`, not `v`?

Also, the polyfill build/instructions would need to add `includes`. If we go through all of our `indexOf(...) !== -1`s and change them, that's worth it, but as it stands, let's just use indexOf here

Reply:
`null` and `undefined` *values* are indeed ignored, since both would be converted to number 0 in `normalized`.

Switched to `indexOf`.